wp-content/themes/twentyseventeen-wp20/functions.php

```diff
@@ -8,12 +8,14 @@
 add_action( 'wp_enqueue_scripts',    __NAMESPACE__ . '\enqueue_scripts', 11              );
 add_filter( 'get_custom_logo',       __NAMESPACE__ . '\set_custom_logo'                  );
 add_filter( 'body_class',            __NAMESPACE__ . '\add_body_classes'                 );
-add_filter( 'the_title',             __NAMESPACE__ . '\internationalize_titles'          );
-add_filter( 'document_title_parts',  __NAMESPACE__ . '\internationalize_document_titles' );
+add_filter( 'the_title',             __NAMESPACE__ . '\internationalize_titles', 11      );
+add_filter( 'the_title',             __NAMESPACE__ . '\prevent_widows_in_content', 12    );
```

Comment:
Didn't realise at the time that this would affect the `title` tag output:

![Screen Shot 2023-04-18 at 10 54 15 AM](https://user-images.githubusercontent.com/1017872/232627607-4af2d0ca-64bf-4ade-88d2-1d758f102b3e.jpg)

Not sure if it matters or not

Reply:
On line 206 below I think we need to update the case to:

```
case '#WP20 Live':
			// translators: The name of the page that displays the #wp20 social media posts in real time.
			$title = esc_html_x( '#WP20 Live', 'adjective', 'wp20' );
			break;
```

Reply:
🤔 , it shouldn't in most cases, but there may be some where it does, like a feedreader, social media site, etc that doesn't convert entities. Fixed in 39d55da